acceptance/lib/helper.rb

```diff
@@ -61,7 +61,9 @@ def self.get_option_value(value, legal_values, description,
 
     # precedence is environment variable, option file, default value
     value = ((env_var_name && ENV[env_var_name]) || value || default_value)
-    if value
+    if value == ""
+      value = default_value
```

Comment:
Hm, yeah, there's probably room for improvement in here but is it worth the trouble?  You could argue that changing all the usages (and missing a few?) is worse than making the one function smarter.